README.md

```diff
@@ -1,8 +1,8 @@
 # rocBLAS-Examples
-Examples for using rocBLAS (Radeon Open Compute Basic Linear Algebra Subprograms) which is a GPU exploiting implementation of BLAS.
+Examples for using rocBLAS which is a GPU exploiting implementation of BLAS.
```

Comment:
Could link directly to rocBLAS doc or github, but ROCm link in next paragraph is probably fine too.

Reply:
Will leave it for now as the link isn't very far down.